src/mixins/name-request-mixin.ts

```diff
@@ -112,6 +116,21 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
     if (nr.state === NameRequestStates.CONDITIONAL) {
       return nr.names.find(name => name.state === NameRequestStates.CONDITION).name
     }
-    return '' // should never happen
+    return null // should never happen
```

Comment:
Both '' and null display "nothing" (so this change doesn't break anything), and I think returning null is more precise if this should never happen anyway.
Both '' and null are falsey, so they're interchangeable (except you can use string methods on '' but not on null).
Any thoughts on which is better or if this matters?